code-challenges.test.js

```diff
@@ -14,13 +14,47 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+// const fibLength1 = 6
+// Expected output: [1, 1, 2, 3, 5, 8]
+
+// const fibLength2 = 10
+// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+describe("fib1", () => {
+  it("returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    expect(fib1(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fib1(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+
+//  FAIL  week-3-assessment-jricks86/code-challenges.test.js
+  // ● fib1 › returns an array that length containing the numbers of the Fibonacci sequence
+
+  // ReferenceError: fib1 is not defined
+
+// b) Create the function that makes the test pass.
+  // input: number
+  // output: array containing the numbers in Fibonacci sequence
+  //Declare an array to store the fibonacci sequence
+ //For Loop through the sequence until the number is reached starting at the 2 index
+ //calculate the next number in the sequence by adding the previous two numbers and push it into the fib array.
```

Comment:
This describes what you want the function to do, but not fully how it's going to do it